packages/cli/src/commands/organization-services/list.ts

```diff
@@ -0,0 +1,43 @@
+import { vaultAPIFactory } from '@meeco/sdk';
+import { cli } from 'cli-ux';
+import { AuthConfig } from '../../configs/auth-config';
+import { OrganizationServiceListConfig } from '../../configs/organization-services-list-config';
+import { authFlags } from '../../flags/auth-flags';
+import MeecoCommand from '../../util/meeco-command';
+
+export default class OrganizationServicesList extends MeecoCommand {
+  static description =
+    'List requested services for a given organization. Members of the organization with roles owner and admin can use this command to list the requested services for this organization.';
```

Comment:
Wrap in backticks and split across multiple lines for readability.

Reply:
done

Reply:
Ah, sorry I meant for slightly tighter line wrapping - sort of like this:

```
  static description = `List requested services for a given organization.
    Members of the organization with roles owner and admin can use this
    command to list the requested services for this organization.`;
```